.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "eslint:recommended",
   "env": {
+    "es6": true,
```

Comment:
are we using any ES6 APIs that require this change?

Reply:
We need at least `"parserOptions": { "ecmaVersion": 6 }` to cover the ES6 syntax such as `const` and shorthand methods.

`es6: true` includes this syntax automatically, though it also allows ES6 globals which the former does not. I can change to the former if you prefer, but just seems easier in case we need the globals later since we're already on ES6.

Reply:
yeah, I think I'd prefer the `parserOptions` approach

Reply:
Ok, changed.

Reply:
And rebased.